docs/content/latest/faq/yugabyte-platform.md

```diff
@@ -14,92 +14,85 @@ isTocNested: false
+For instructions on installing Yugabyte Platform, refer to [Install Yugabyte Platform](../../yugabyte-platform/install-yugabyte-platform/).
 
-- Ubuntu 16.04+
-- Red Hat Enterprise Linux 6.5+
-- CentOS 7+
-- Amazon AMI 2014.03 / 2014.09 / 2015.03 / 2015.09 / 2016.03 / 2016.09
+## What are the OS requirements and permissions to run Yugabyte Platform?
 
-For the complete list of operating systems supported by Replicated, see [Supported OS](https://www.replicated.com/docs/distributing-an-application/supported-operating-systems/).
+Yugabyte Platform requires Replicated; currently, only Linux-based operating systems are supported by Replicated. This Linux OS should be 3.10+ kernel, 64-bit, and ready to run docker-engine 1.7.1 - 17.06.2-ce (with 17.06.2-ce being the recommended version).
+
+For a complete list of operating systems supported by Replicated, see [Supported Operating Systems](https://help.replicated.com/docs/native/customer-installations/supported-operating-systems/).
 
 {{< note title="Note" >}}
 
-The preceding list of operating systems is only applicable to Yugabyte Platform. For a list of OSs supported by YugabyteDB, see [Prerequisites for Installing YugabyteDB](https://docs.yugabyte.com/latest/quick-start/install/linux/#prerequisites).
+This requirement applies only to Yugabyte Platform. For a list of OSs supported by YugabyteDB, see the [Deployment checklist](../../../deploy/checklist/) for YugabyteDB.
 
 {{< /note >}}
 
-**Permissions necessary for an Internet-connected host**
-
-- Connectivity to the Internet, either directly or via an HTTP proxy
-- Ability to install and configure [docker-engine](https://docs.docker.com/engine/)
-- Ability to install and configure [Replicated](https://www.replicated.com/), which is a containerized application itself and needs to pull containers from its own Replicated.com container registry
-- Ability to pull Yugabyte container images from [Quay.io](https://quay.io/) container registry, this will be done by Replicated automatically
-
-**Permissions necessary for an airgapped host**
-
-An _airgapped_ host has no path to inbound or outbound Internet traffic at all. For such hosts, the installation is performed as a `sudo` user.
-
-**Additional requirements**
+Yugabyte Platform also requires the following:
 
-For airgapped hosts, a supported version of docker-engine (currently 1.7.1 to 17.03.1-ce). If you do not have docker-engine installed, follow the instructions [here](https://www.replicated.com/docs/kb/supporting-your-customers/installing-docker-in-airgapped/) to first install docker-engine.
+- Connectivity to the Internet, either directly or via an HTTP proxy.
+- Ability to install and configure [docker-engine](https://docs.docker.com/engine/).
+- Ability to install and configure [Replicated](https://www.replicated.com/), which is a containerized application itself and needs to pull containers from its own Replicated.com container registry.
```

Comment:
Maybe a good idea to point to the [Replicated.com container registry](https://help.replicated.com/docs/native/getting-started/docker-registries/). I am guessing this is the one.